_content/articles/pali-buddhist-studies-in-japan_ota-kiyoshi-ikeda-masataka.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Pali Buddhist Studies in Japan"
+authors:
+  - "Kiyoshi Ota and Masataka Ikeda"
+external_url: "https://doi.org/10.1558/bsrv.v6i1.13965"
```

Comment:
Move the DOI to source and use this for the external_url https://archive.org/download/pali-buddhist-studies-in-japan-1981/Pali%20Buddhist%20Studies%20in%20Japan%20%281981%29_text.pdf